### PR TITLE
Feature: Add an option to make bridge branches in makeLODF

### PR DESCRIPTION
**Pull request by determ1ne**
See #182 for details.

### CODE REVIEW

File: lib/makeLODF.m [hunk 1]
```diff
@@ -1,23 +1,51 @@
...
+if nargin < 3
+    mask_bridge = 0;
+else %% nargin >= 3
+    if ~exist('mpc', 'var')
+        error('MPC is required when mask_bridge is enabled.');
+    end
+end
+if ~exist('mask_bridge', 'var')
+    mask_bridge = 0;
+end
```
Comment (rdzman) [inline lib/makeLODF.m:47]: Isn't case already included in lines 38-39?

Comment (determ1ne) [reply lib/makeLODF.m]: You're right, I've removed them.

**Comment by determ1ne** [comment]
Any update on this?

**Comment by rdzman** [comment]
Thanks for the contribution @determ1ne!

**Pull request merged**